tests/functional/spec/dob_date.spec.js

```diff
@@ -0,0 +1,25 @@
+import DateOfBirthPage from "../generated_pages/dob_date/date-of-birth.page";
+import UnderSixteenPage from "../generated_pages/dob_date/under-sixteen.page";
+
+describe("Date of birth check", () => {
+  beforeEach("Load the survey", () => {
+    browser.openQuestionnaire("test_dob_date.json");
+  });
+  it("Given when I start survey, Then I see date of birth page", () => {
```

Comment:
Not sure this adds much, as date of birth isn't a type and therefore isn't doing anything special in regards to legends